examples/fluid/taylor_green_vortex_2d.jl

```diff
@@ -58,13 +58,16 @@ background_pressure = sound_speed^2 * fluid_density
 smoothing_length = 1.0 * particle_spacing
 smoothing_kernel = SchoenbergQuinticSplineKernel{2}()
 
+pertube_coordinates = true
 fluid = RectangularShape(particle_spacing, (n_particles_xy, n_particles_xy), (0.0, 0.0),
-                         coordinates_perturbation=0.2, # To avoid stagnant streamlines when not using TVF.
+                         coordinates_perturbation=pertube_coordinates ? 0.2 : nothing, # To avoid stagnant streamlines when not using TVF.
```

Comment:
```suggestion
                         # Perturb particle coordinates to avoid stagnant streamlines without TVF.
                         coordinates_perturbation=pertube_coordinates ? 0.2 : nothing,
```